### PR TITLE
Add cross device url enterprise feature

### PR DESCRIPTION
**Pull request by eraxyz**
# Problem

Enterprise customers need the ability to change the url used for the cross device flow.

# Solution

If a `cross_device_url` is found in the SDK Token, use it to initiate the cross device flow instead of the Onfido default.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [X] Has the CHANGELOG been updated?
- [X] Has the README been updated?
- [n/a] Has the CONTRIBUTING doc been updated?
- [n/a] Has the RELEASE_GUIDELINES been updated?
- [n/a] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [n/a] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [n/a] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1052-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Added basic history to SDK demo.
 - Public: Added French translation. The language tag is `fr_FR`.
 - UI: Added passport quality guide before upload/capture.
+- Public: Added check for cross_device_url in SDK Token to be used as the link for all cross device options instead of the default HOSTED_SDK_URL.
```
Comment (kopijunkie) [inline CHANGELOG.md:14]: I think we should make it clear here also it's an enterprise feature and short explanation of what cross_device_url is.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1052-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md (hunk 1 above)
Comment (eraxyz) [reply CHANGELOG.md]: Thanks, added some more detail

File: README.md [hunk 2]
```diff
@@ -72,6 +72,18 @@ Make a note of the `token` value in the response, as you will need it later on w
...
+  ### Cross device URL ###
+
+  This is an enterprise feature that must be enabled for your account before it can be used. Once enabled you will be able to specify your own custom url that the cross-device flow will redirect to instead of the Onfido default. To use this feature generate a SDK token as shown below and use it to start the SDK.
+
+   ```shell
+  $ curl https://api.onfido.com/v3/sdk_token \
+    -H 'Authorization: Token token=YOUR_API_TOKEN' \
+    -F 'applicant_id=YOUR_APPLICANT_ID' \
+    -F 'referrer=REFERRER_PATTERN' \
+    -F 'cross_device_url=YOUR_CUSTOM_URL'
```
[the remark this answers is not in the document]
Comment (eraxyz) [reply README.md:84]: Not done yet but coordinating with Ben and CAE team to get this done soon.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1052-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1052-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**